common/default.nix

```diff
@@ -0,0 +1,24 @@
+{ mkDerivation, aeson, async, attoparsec, base, bytestring
```

Comment:
Is this still needed? I don't think that obelisk dependency needs to be removed; I think there was some confusion in the past because `obelisk-asset` used to be a thunk.

Reply:
I think monad-logger needs it otherwise Android stuff doesn't work

Reply:
Oh I see this already exists on e.g. `tezos-reuification`. Good for now +1.